wrapping/python/openmeeg/openmeeg.i

```diff
@@ -527,6 +560,7 @@ namespace OpenMEEG {
 %extend OpenMEEG::Geometry {
 
     Geometry(PyObject* pylist) {
+        std::cout << "Create Geometry from list of lists" << std::endl;
```

Comment:
... which means you end up in this code path, not any that deal with any OpenMEEG `Mesh` at the Python level. 

> def _mesh_vertices_and_triangles(mesh):

Let's just get rid of this if it's unsafe. The failing code path (or at least one of them) uses the list-of-list-of-ndarray approach, not this code, anyway. I'll push a commit to make it clearer that this is not used.

So one of the failures is exactly with the code path you suggest, I think:

> I did the interface with the idea that mne directly wants to provide its own meshes in terms of vertices and triangles.
>
> ... but one thing that strikes me is the absence of indmap.

In this code path, `indmap` is created in C++ in the loop below. If you look at the traceback of the failure, you can even see that it's in this code path, because it reflects the messages printed below, stitched back together in the order in which they actually occurred:

```
C:\Miniconda\envs\test\lib\site-packages\openmeeg\tests\test_python.py:136: in test_python
    geom = om.make_nested_geometry(meshes, conductivity=(1, 0.0125, 1))
C:\Miniconda\envs\test\lib\site-packages\openmeeg\_make_geometry.py:203: in make_nested_geometry
    geom = make_geometry(meshes, interfaces, domains)
C:\Miniconda\envs\test\lib\site-packages\openmeeg\_make_geometry.py:69: in make_geometry
    geom = Geometry(meshes)
C:\Miniconda\envs\test\lib\site-packages\openmeeg\openmeeg.py:1224: in __init__
    _openmeeg.Geometry_swiginit(self, _openmeeg.new_Geometry(*args))

Create Geometry from list of lists
    geom_add_vertices(geom, pyobj) done.
    geom_add_vertices(geom, pyobj) done.
    geom_add_vertices(geom, pyobj) done.
    mesh_add_triangles for mesh Cortex : reference_vertices : triangles : check done.
Mesh Cortex (000001FDCBC6F070) consistent during SWIG Geometry update of meshes(0)="Cortex"
    mesh_add_triangles for mesh Skull : reference_vertices : triangles : check done.

E   RuntimeError: Unknown vertex in triangle: Mesh Cortex invalid    during SWIG Geometry update of meshes(1)="Skull", requested triangle vertex address:
E     000001FDCE2F8B80 (000001FDCE1D1060)
E   but valid triangle vertex addresses are:
E     000001FDCE1D1A60
E     000001FDCE1D1880
E     000001FDCE1D19C0
```
So don't look at the `test_make_geometry_head` failures, but rather the `test_python[True-Head1]` failures, like:

https://github.com/openmeeg/openmeeg/runs/8191980344?check_suite_focus=true#step:20:541

In the first iteration where Cortex is added, it's fine. But then once Skull is added, Cortex is broken.